shared/src/commonTest/kotlin/vn/luongvo/kmm/survey/data/local/datasource/TokenLocalDataSourceTest.kt

```diff
@@ -28,13 +28,13 @@ class TokenLocalDataSourceTest {
         dataSource.saveToken(token)
 
         verify(mockSettings)
-            .invocation { set("tokenType", "tokenType") }
+            .invocation { set("tokenType", "Bearer") }
             .wasInvoked(exactly = 1.time)
         verify(mockSettings)
-            .invocation { set("accessToken", "accessToken") }
+            .invocation { set("accessToken", "lbxD2K2BjbYtNzz8xjvh2FvSKx838KBCf79q773kq2c") }
```

Comment:
I think it's better to use a more generic data for test or there is now a new limitation?🤔 
```suggestion
            .invocation { set("accessToken", ["accessToken")](url) }
```

Reply:
@blyscuit I verify that value to the one in the [mock json response](https://github.com/luongvo/kmm-survey/blob/c52383e6aafc610e44d4854e8e73126c7b3bf6f1/shared/src/commonTest/resources/oauth_login.json#L6). It was long long time ago in my Flutter IC so it's safe ✅